python-modules-list.sh

```diff
@@ -1,8 +1,12 @@
 package: Python-modules-list
 version: "1.0"
 env:
-  PIP_REQUIREMENTS: |
+  PIP_BASE_REQUIREMENTS: |
+    pip==23.0.1
+    setuptools==65.5.1
+    wheel
     requests==2.27.1
```

Comment:
Why is requests in `PIP_BASE_REQUIREMENTS`? Shouldn't it be in `PIP_REQUIREMENTS`, as before?

Reply:
well, requests is used on any http communication, and i seen that is not installed on osx_arm64 so i thought this is a forgotten dependency .. so, should it be moved back to PIP_REQUIREMENTS?